app/database/migrations/20210210230058_add_name_and_instructor_columns.js

```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex) {
+    knex.schema.table('jobs', function(table){
```

Comment:
Should we also store the start time of the class?

Reply:
oh this is just adding more columns to the table because we needed some more info for the front end, so all the old stuff is still there.

Reply:
Yeah, I was just wondering if there were any other columns we needed to add. I thought in the previous migrations we added a column for the job's scheduled time and not the class, but I guess that isn't necessary as they are related.